h/search/parser.py

```diff
@@ -16,7 +16,7 @@
 pp.ParserElement.enablePackrat()
 
 # Named fields we support when querying (e.g. `user:luke`)
-named_fields = ['user', 'tag', 'group', 'uri', 'url']
+named_fields = ['user', 'tag', 'group', 'uri', 'url', 'wildcard_uri']
```

Comment:
It might be convenient to support "site:foo.com" or "domain:foo.com" as a shortcut for "httpx://foo.com/*" (ie. all pages on HTTP or HTTPS foo.com URLs). The former is what Google uses for searching eg. all pages on a particular domain.